src/sentry/integrations/vercel/client.py

```diff
@@ -35,3 +37,12 @@ def get_user(self):
     def get_projects(self):
         # TODO: we will need pagination since we are limited to 20
         return self.get(self.PROJECTS_URL)["projects"]
+
+    def create_deploy_webhook(self):
+        data = {
+            "name": "sentry_webhook",
```

Comment:
@scefali don't have a strong attachment to this name, so we can change it if we want

Reply:
@MeredithAnya Maybe "Sentry webhook"? The examples I see in their docs use spaces and capital letters.